dask_cudf/tests/test_groupby.py

```diff
@@ -91,7 +97,7 @@ def test_groupby_multi_column(func):
 
     ddf = dask_cudf.from_cudf(gdf, npartitions=5)
 
-    a = func(gdf).to_pandas()
-    b = func(ddf).compute().to_pandas()
+    a = pd.DataFrame(func(gdf).to_pandas())
+    b = pd.DataFrame(func(ddf).compute().to_pandas())
```

Comment:
Instead of doing this I would just xfail the parameter above for now. We should fix it in the future.